packages/earth-admin/src/components/sidebar-select/model.ts

```diff
@@ -20,6 +20,7 @@
 import { AuthzGuards } from 'auth/permissions';
 
 export const ADMIN_PAGES: IAdminPage[] = [
+  { key: 'Home', url: '', guard: AuthzGuards.accessPlacesGuard },
```

Comment:
is this the correct auth?

Reply:
`AuthzGuards.readAllGuard` is probably a better fit.